### ISSUE TITLE
I can't believe const reassign is alright from loopbacks point of view

### ISSUE THREAD

**Issue by viktor-ku** [strongloop/eslint-config-loopback#35]
Hey! It's just a quick question: is it intentional or why otherwise loopback config doesn't catch this

```js
const foo = 3;
foo = 4;
```

Thanks
